scrapers/VlogXXX.yml

```diff
@@ -0,0 +1,27 @@
+name: VlogXXX
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - vlogxxx.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="row"]/h1
+      Performers:
+        Name: //h3[contains(.,"pornstars")]/following-sibling::a
+      Date:
+         selector: //p[@class="date"]
+         postProcess:
+           - parseDate:  2006-01-02
+      Tags:
+        Name: //h3[contains(.,"Categories")]/following-sibling::a
+      Details:
+        selector: //div[@class="col-12 col-md-6 line"]/p
```

Comment:
since you are using text for the tags and the performers
this 
```yaml
        selector: //div[h3[text()="Description"]]/p
```
looks better :P

Reply:
Dressed better now :P